mars/services/task/execution/ray/fetcher.py

```diff
@@ -28,6 +30,8 @@ class RayFetcher(Fetcher):
     required_meta_keys = ("object_refs",)
 
     def __init__(self, **kwargs):
+        _make_query_function_remote()
```

Comment:
It's not a good idea to check `query_object_with_condition` instance every time when the RayFetcher constructing.

Reply:
Can we use another object `_query_object_with_condition` for the @remote decorated function.

Reply:
> query_object_with_condition

Will use None-check instead.